src/main/java/filter/expression/Qualifier.java

```diff
@@ -853,16 +855,21 @@ private boolean assigneeSatisfies(IModel model, TurboIssue issue) {
         return login.contains(content) || name.contains(content);
     }
 
-    private boolean authorSatisfies(TurboIssue issue) {
+    private boolean authorSatisfies(IModel model, TurboIssue issue) {
         if (!content.isPresent()) return false;
 
+        boolean shouldWarnUser = !model.isUserInRepo(issue.getRepoId(), content.get());
+        if (shouldWarnUser) {
+            throw new SemanticException(String.format(USER_WARNING_ERROR_FORMAT, content.get(), issue.getRepoId()));
+        }
```

Comment:
This code block seems similar to another block above. If they are, may be extract into a method?